api/README.md

```diff
@@ -48,32 +50,53 @@ To authenticate to UID2 endpoints, use a bearer token in the request's authoriza
 
 ```Authorization: Bearer YourTokenBV3tua4BXNw+HVUFpxLlGy8nWN6mtgMlIk=```
 
+## Query Parameter Value Encoding
+
+When passing query parameter values in a request, ensure the query parameter value is URL-encoded. Use JavaScript's `encodeURIcomponent()` or its equivalent in your coding language.
+
 ## Email Address Normalization
 
->NOTE: The UID2 Operator Service normalizes automatically unhashed email addresses. Hashed email adresses must be normalized.
+The UID2 Operator Service normalizes unhashed email addresses automatically. You must normalize hashed email adresses before sending them in a request.
 
-Prior to sending email addresses in a request, normalize them by following these steps:
+To normalize hashed email addresses, complete the following steps:
 
 1. Remove leading and trailing spaces.
 2. Convert all ASCII characters to lowercase.
 3. In `gmail.com` email addresses, remove the following characters from the username part of the email address:
     1. The period  (`.` (ASCII code 46)).<br/>For example, normalize `jane.doe@gmail.com` to `janedoe@gmail.com`.
     2. The plus sign (`+` (ASCII code 43)) and all subsequent characters.<br/>For example, normalize `janedoe+home@gmail.com` to `janedoe@gmail.com`.
 
-## Query Parameter Value Encoding
-
-When passing query parameter values in with a request, ensure the query parameter value is URL-encoded. Use JavaScript's `encodeURIcomponent()` or its equivalent in your coding language.
-
 ## Email Address Hash Encoding
 
 Email hashes are base64-encoded SHA256 hashes of the normalized email address.
 
-| Type | Example | Use |
+| Type | Example | Usage |
+| :--- | :--- | :--- |
+| Normalised email address | `user@example.com` | |
+| SHA256 of email address | `b4c9a289323b21a01c3e940f150eb9b8c542587f1abfd8f0e1cc1ffc5e475514` | |
+| base64-encoded SHA256 of email address | `tMmiiTI7IaAcPpQPFQ65uMVCWH8av9jw4cwf/F5HVRQ=` | Use this encoding for `email_hash` values sent in the request body. |
+| URL-encoded, base64-encoded SHA256 of email address| `tMmiiTI7IaAcPpQPFQ65uMVCWH8av9jw4cwf%2FF5HVRQ%3D` | Use this encoding for `email_hash` query parameter values. |
+
+## Phone Number Normalization
+
+>IMPORTANT: You must normalize both hashed and unhashed phone numbers before sending them in a request.
+
+Here's what you need to know about phone number normalization rules:
+
+- The UID2 Operator accepts phone numbers in the [E.164](https://en.wikipedia.org/wiki/E.164) format, which is the international telephone number format that ensures global uniqueness. 
+- E.164 phone numbers can have a maximum of 15 digits.
+- Normalized E.164 phone numbers use the following syntax: `[+] [country code] [subscriber number including area code]`, with no spaces, hyphens, parentheses, or other characters. For example, the phone numbers `+123 44 555-66-77` and `1 (123) 456-7890` would be normalized as `+123445556677` and `+11234567890`, respectively.
```

Comment:
`would be normalized` -> `must be normalized`?

Reply:
Updated as suggested.